examples/feature_control_examples/public/app.tsx

```diff
@@ -52,7 +52,7 @@ export const MyPluginComponent: React.FC = () => {
         ).map(([capability, value]) => {
           return value === true ? (
             <div key={capability}>
-              <EuiHealth color="success">{capability}</EuiHealth>
+              <EuiHealth color="accentSecondary">{capability}</EuiHealth>
```

Comment:
@SiddharthMantri Upon checking with the EUI team, it appears we won't have `accentSecondary` as an option for **EuiHealth**. Without having seen the screenshot for this I imagine this **EuiHealth** element is trying to convey something positive, so staying with the `success` color here would be the recommended course of action.

Reply:
Thanks @andreadelrio! Fixed in https://github.com/elastic/kibana/pull/201795/commits/9ef41118bcf15329542403254ef28382cf511a76